src/libraries/Microsoft.PowerFx.Interpreter/Functions/StandardErrorHandling.cs

```diff
@@ -18,7 +18,8 @@ private enum ReturnBehavior
             AlwaysEvaluateAndReturnResult,
             ReturnBlankIfAnyArgIsBlank,
             ReturnEmptyStringIfAnyArgIsBlank,
-            ReturnFalseIfAnyArgIsBlank
+            ReturnFalseIfAnyArgIsBlank,
+            ReturnBlankIfOneArgIsBlankAndEmptyStringForOtherArgIsBlank
```

Comment:
>ReturnBlankIfOneArgIsBlankAndEmptyStringForOtherArgIsBlank [](http://example.com/codeflow?start=12&length=58)

This is way too specific - it's only used by the Text function. Do we need to have a new one, or should just use AlwaysEvaluateAndReturnResult and let the Text function handle it?

Reply:
Updated here and handle in text.